src/databricks/labs/ucx/install.py

```diff
@@ -812,6 +812,28 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def repair_run(self, workflow):
+        try:
+            job_id = self._state.jobs.get(workflow)
+            if not job_id:
+                logger.warning(f"{workflow} job does not exists hence skipping Repair Run")
+                return
+            job_runs = list(self._ws.jobs.list_runs(job_id=job_id, limit=1))
+            if not job_runs:
+                logger.warning(f"{workflow} job is not initialized yet. Can't trigger repair run now")
+                return
+            latest_job_run = job_runs[0]
+            state = latest_job_run.state
+            if state.result_state.value != "SUCCESS":
+                run_id = latest_job_run.run_id
+                job_run_waiter = self._ws.jobs.repair_run(run_id=run_id, rerun_all_failed_tasks=True)
+                job_run_waiter.result()
```

Comment:
This would timeout in 20 minutes. What you should do instead is construct a url of this new run and open it in browser and return from cli

Reply:
removed calling the result.Instead opening the run url in browser.